mythril/laser/smt/bitvec_helper.py

```diff
@@ -103,7 +103,7 @@ def ULT(a: BitVec, b: BitVec) -> Bool:
 
 
 def ULE(a: BitVec, b: BitVec) -> Bool:
-    """Create an unsigned less than expression.
+    """Create an unsigned less or equals than expression.
```

Comment:
Nice catch! can you switch it to `less than or equal to expression`

Reply:
Okay